src/renderer/stores/game.ts

```diff
@@ -22,10 +21,6 @@ export default class GameStore {
     this._appProtocolVersion = electronStore.get(
       "AppProtocolVersion"
     ) as string;
-
-    if (!(this._language in supportedLocales)) {
-      this._language = "en";
-    }
```

Comment:
We need to leave this option for Unity Game. Can't we check the supported locale from Transifex?

Reply:
I haven't found any validation functionality provided by Transifex. I'll check if any other third-party libraries can help.

I doubt that this should be done in simple equality check. For instance, if the user locale is `en_GB` it won't be equal to `en` or `en_US`. Although this exact case won't be the problem since English is the default.

Reply:
Yes, but at least I knew, the game needs default language and it's only `en`, do not need detail like `en_US`.

Reply:
What I meant is some users may experience fallback to English even if their language is supported on the game. Maybe we could address this one in detail in next meetup.

Reply:
The validation step is moved to the provider's effects, and the GameStore should be able to update its language by the added snippet subscribing to locale changes.